src/Redux/actionCreators/ServiceActions.js

```diff
@@ -121,3 +122,41 @@ export const resetFilter = ({ pagination }) => dispatch => {
     .then(() => dispatch(loaderActions.stopAIServiceListLoader))
     .catch(() => dispatch(loaderActions.stopAIServiceListLoader));
 };
+
+const fetchFeedbackAPI = (username, orgId, serviceId, token) => {
+  const apiName = APIEndpoints.USER.name;
+  const path = `${APIPaths.FEEDBACK}?username=${username}&org_id=${orgId}&service_id=${serviceId}`;
+  const myInit = {
```

Comment:
Can we have one method that constructs myInit? The method can take in header values and body value and return this object. I see that its used in other API calls as well

Reply:
resolved in [commit](https://github.com/singnet/refactored-dapp/pull/109/commits/56e18282bd7504a6989b5ea1d22e9c4b581a313f)